central/graphql/resolvers/clusters.go

```diff
@@ -553,6 +557,39 @@ func (resolver *clusterResolver) VulnCounter(ctx context.Context, args RawQuery)
 	return resolver.root.VulnCounter(ctx, RawQuery{Query: &query})
 }
 
+func (resolver *clusterResolver) NodeVulns(ctx context.Context, args PaginatedQuery) ([]NodeVulnerabilityResolver, error) {
+	defer metrics.SetGraphQLOperationDurationTime(time.Now(), pkgMetrics.Cluster, "NodeVulns")
+
+	if !features.PostgresDatastore.Enabled() {
+		query := search.AddRawQueriesAsConjunction(args.String(), resolver.getClusterRawQuery())
```

Comment:
Let's make the sub-resolver calls uniform. For all calls to sub-resolvers use `scoped.Scope`. You will find such examples in image, and deployments.

Reply:
Cluster scoping the context is not able to resolve node vulns when combined with `Fixable : true` or `Fixable : false` filters. Created [ROX-10911](https://issues.redhat.com/browse/ROX-10911) to track.